purk/irc.py

```diff
@@ -246,16 +245,15 @@ def on_connect(self, result, error):
             self.core.run_command("/join %s" % channelother)
 
     # called when we read data or failed to read data
-
     def on_read(self, result, error):
         if error:
-            self.disconnect(error=error[1])
+            self.disconnect(error=strerror)
```

Comment:
I guess L250 will raise a `NameError`. Possibly, it would be a typo, which might be actually `error.strerror`